Keas.Mvc/ClientApp/components/Access/AssignAccess.tsx

```diff
@@ -74,7 +74,7 @@ export default class AssignAccess extends React.Component<IProps, IState> {
                 >
                     <div className="modal-header row justify-content-between">
                         <h2>{this.props.selectedAccess || this.props.person ? "Assign Access" : "Add Access"}</h2>
-                        <Button color="link" onClick={this._closeModal}>
+                        <Button color="link" onClick={this._closeModal_using_icon}>
```

Comment:
@NabilFurmoli try to keep the naming on your functions consistent. The original function is private, so it's `_closeModal`, so your new function should be something like `_closeModalUsingIcon`, or name your new function `_confirmCloseModal`. Your new function might also just call `_closeModal` itself after confirming, so that you don't have reused logic.

Reply:
Ok sounds good I will update the function names.